.travis.yml

```diff
@@ -163,14 +251,13 @@ jobs:
             - libc6-dev-i386
       before_install:
         - mkdir bin ; ln -s /usr/bin/gcc-5 bin/gcc
-        - export CCACHE_CPP2=yes
       # env: COMPILER=clang++-3.7 SAN_FLAGS="-fsanitize=undefined -fno-sanitize-recover=undefined,integer -fno-omit-frame-pointer"
       env:
         - COMPILER="ccache /usr/bin/clang++-3.7"
         - EXTRA_CXXFLAGS="-Qunused-arguments -fcolor-diagnostics -DNDEBUG"
         - CCACHE_CPP2=yes
 
-    # Ubuntu Linux with glibc using clang++-3.7, debug mode, disable USE_DSTRING
+    # Ubuntu Linux with glibc using clang++-3.7, debug mode
     - stage: Test different OS/CXX/Flags
```

Comment:
Should these 'build only' tests (i.e. the debug builds) be either moved into the first stage (i.e. builds), or positioned into a third 'debug builds' stage? I don't have a strong opinion, but thought it was worth us making an explicit decision.